keps/sig-storage/20200609-csi-service-account-token.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Service Account Token for CSI Driver
+authors:
+  - "@zshihang"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
+  - sig-storage
+reviewers:
+  - "@msau42"
+approvers:
+creation-date: 2020-06-09
+last-updated: 2020-06-09
+status: implementable
+---
+
+# Service Account Token for CSI Driver
+
+## Table of Contents
+
+<!-- toc -->
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [User stories](#user-stories)
+- [Proposal](#proposal)
+  - [CSIDriver Object](#csidriver-object)
+  - [Example Workflow](#example-workflow)
+- [Alternatives](#alternatives)
+- [Implementation History](#implementation-history)
+
+<!-- /toc -->
+
+## Summary
+
+This KEP proposes a way to obtain service account token for pods that the CSI
+drivers are provisioning volumes for. Since these tokens are valid only for a
```

Comment:
The first sentence implies that the token is available in CreateVolume, while the token is passed in NodePublish (which makes more sense).

Reply:
will reword it. :)